cmd/contour/leadership.go

```diff
@@ -133,6 +136,8 @@ func newResourceLock(ctx *serveContext, clients *k8s.Clients) resourcelock.Inter
 			Identity: resourceLockID,
 		},
 	)
-	check(err)
+	if err != nil {
```

Comment:
Could you please change `newResourceLock` to take a `log logrus.FieldLogger` param? Then we no longer need to import `logrus` twice.